gammapy/obs/observation.py

```diff
@@ -11,7 +14,7 @@
 
 
 class Observation(object):
-    """Observation (a.k.a. Run).
+    """Observation (a.k.a. run).
```

Comment:
If you look here: https://gammapy.readthedocs.org/en/latest/obs/index.html#classes
you'll see that in the short description of the class the line is truncated:
```
Observation (a.k.a.
```
This is because Sphinx or the Numpy docstring Sphinx extension looks for a dot `.` when parsing the docstring.

So the one-line description should always be ended with a dot, and can't have dots inside.
So please rephrase here, e.g. `"""A single observation."""` and then explain later that observation is called run by some people if you think it's useful.
This class might go away, so it's not super-important.

I'm just mentioning this here because I want you to become a Gammapy / Python / Sphinx pro and over the coming weeks help clean up and improve the code and docs where-ever you see an opportunity for improvement. We can only achieve a really nice package if we keep re-factoring and polishing and we need your help with that.